<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainview</class>
 <widget class="QMainWindow" name="mainview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>社区医疗管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabDoctorMessage">
       <attribute name="title">
        <string>医生信息</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>查找内容：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="SearchDoctorEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="SearchDoctorButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>查询</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="SortDoctorlabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>排序：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="SortDoctorcomboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>按年龄升序</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>按年龄降序</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>按身高升序</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>按身高降序</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>按体重升序</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>按体重降序</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>按入职时间升序</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>按入职时间降序</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ActionDoctorlabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>操作：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="ActionDoctorcomboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>录入</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>修改</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>删除</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>导入</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>导出</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="DoctorMessageView"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPatientMessage">
       <attribute name="title">
        <string>患者信息</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>查找内容：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SearchPatientEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SearchPatientButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="SortPatientlabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>排序：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="SortPatientcomboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>按年龄升序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按年龄降序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按身高升序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按身高降序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按体重升序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按体重降序</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ActionPatientlabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>操作：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ActionPatientcomboBox">
            <item>
             <property name="text">
              <string>录入</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>修改</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>删除</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>导入</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>导出</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="PatientMessageView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDrugMessage">
       <attribute name="title">
        <string>药品信息</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>查找内容：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SearchDrugEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SearchDrugButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="SortDruglabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>排序：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="SortDrugcomboBox">
            <item>
             <property name="text">
              <string>按库存数量升序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按库存数量降序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按保质期升序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按保质期降序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按入库时间升序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按入库时间降序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按出库时间升序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按出库时间降序</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ActionDruglabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>操作：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ActionDrugcomboBox">
            <item>
             <property name="text">
              <string>录入</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>修改</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>删除</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>导入</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>导出</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>库存预警</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="DrugMessageView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTreatRecord">
       <attribute name="title">
        <string>就诊记录</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>查找内容：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SearchTreatRecordEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SearchTreatRecordButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="SortTreatRecordlabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>排序：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="SortTreatRecordcomboBox">
            <item>
             <property name="text">
              <string>按就诊日期升序</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>按就诊日期降序</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="FilterTreatRecordlabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>过滤：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="FilterTreatRecordcomboBox">
            <item>
             <property name="text">
              <string>近一周的就诊记录</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>近一个月的就诊记录</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>近一年的就诊记录</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ActionTreatRecordlabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>操作：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ActionTreatRecordcomboBox">
            <item>
             <property name="text">
              <string>录入</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>修改</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>删除</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>导出</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="TreatRecordMessageView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
